Header header
float64[] centerX
float64[] centerY
float64[] radius
float64[] range
float64[] bearing
int32[] landmarkIndex
int32 landmarkCount
